src/main/java/com/gamedoora/backend/studioservices/assembler/StudioServicesAssembler.java

```diff
@@ -128,7 +128,7 @@ public List<StudiosDTO> getAllStudiosByUsers(long id){
 
     public List<StudiosDTO> getAllStudiosByUsersFirstName(String name){
         List<StudiosDTO> studiosDto = new ArrayList<>();
-        studioRepository.findByUsersSet_FirstName(name).forEach(studio -> studiosDto.add(getStudioMapper().studiosToStudiosDto(studio)));
+        studioRepository.findByName(name).forEach(studio -> studiosDto.add(getStudioMapper().studiosToStudiosDto(studio)));
```

Comment:
Please explain this change? Also are you using the latest updates from the studio service repository?